pepper-apis/studybuilder-cli/studies/osteo/study-events.conf

```diff
@@ -706,8 +706,10 @@
       },
       "action": {
         "type": "SENDGRID_EMAIL",
-        "emailTemplate": ${emails.joinMailingList},
-        "language": "en",
+        "templates": [
+          { "emailTemplate": ${emails.joinMailingList}, "language": "en", "isDynamic": true },
```

Comment:
Existing "en" email templates are not dnamic templates. They are old legacy templates.
"isDynamic" should be false for all existing "en" (english) templates.

Reply:
I see. I will fix this asap.